server/auth_server.go

```diff
@@ -365,6 +552,79 @@ func UpgradeSettings() error {
 	return nil
 }
 
+func UpgradeCase() error {
```

Comment:
This logic will need to change in order to save all the providers, no just the current one